buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/DetermineBaselines.kt

```diff
@@ -87,6 +90,15 @@ open class DetermineBaselines @Inject constructor(isDistributed: Boolean) : Defa
         return commitBaseline(forkPointCommit)
     }
 
+    private
+    fun tryGetUpstream(): String? = project.execAndGetStdout("git", "remote", "-v")
+        .lines()
+        .find { it.contains("git@github.com:gradle/gradle.git") }
```

Comment:
```suggestion
        .find { it.contains("git@github.com:gradle/gradle.git") || it.contains("https://github.com/gradle/gradle.git") }
```

I use https locally because proxies works better for https (yes, name/password can be cached locally so I don't need to input password every time).